dill/_dill.py

```diff
@@ -2117,24 +1868,21 @@ def save_property(pickler, obj):
 @register(classmethod)
 def save_classmethod(pickler, obj):
     log.info("Cm: %s" % obj)
-    im_func = '__func__' if PY3 else 'im_func'
+    im_func = '__func__'
     try:
         orig_func = getattr(obj, im_func)
     except AttributeError:  # Python 2.6
```

Comment:
this is marked as python 2.6, so it might be able to go away.  Not sure.

Reply:
From [this old SO post](https://stackoverflow.com/a/1677671/3738764), it seems that `classmethod`s didn't have a `__func__` attribute before Py2.7. I think it's safe to remove.

Reply:
Actually I remember that now. :)